ivy/functional/frontends/jax/__init__.py

```diff
@@ -7,3 +7,5 @@
 from . import numpy
 from . import random
 from . import _src
+
+frontend_array = numpy.array
```

Comment:
let's make `frontend_array` variable private `_frontend_array`, so users aren't allowed to use that.